### DISCUSSION TITLE
AI科技輔助肺部結節判讀 早期肺癌無所遁形

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS3073-C/Python-Community-GS3073-C#157] [General]
倍利科技與新竹台大分院今天（1日）簽訂「精準智慧醫療合作備忘錄」，新竹台大分院將導入倍利科技的「肺部影像輔助判讀軟體」智慧醫療平台，以AI創新科技輔助肺部小結節判讀，加快肺部結節判讀速度、減少失誤，讓早期肺癌無所遁形，雙方並將共同朝向更多影像AI研發。
https://health.ltn.com.tw/article/breakingnews/3722216

**Comment by bonny123865** [comment]
死亡數高的肺癌能及早發現跟治療，不僅對於病患有很大的幫助，運用科技來輔助判斷更讓智慧醫療方面有很棒的進展!!

**Comment by Timmy110403540** [comment]
希望能利用此技術來減少肺炎或肺癌的發生機率

**Comment by briankuo123** [comment]
台灣很多地方空氣都不是很好，有了這個我們就可以幫助更多病人去解決問題了。

**Comment by wh20502** [comment]
我覺得判斷肺炎固然重要，如果可以在火力發電廠附近加設ai判斷是否有超排，或是排放太多，可能會對附近居民更友善。

**Comment by ghost** [comment]
台灣吸菸人口這麼多 這件事真是這些人的一大福祉

**Comment by lx3211** [comment]
早點發現病症就能及早治療，可以避免很多憾事～

**Comment by NcuJerryKao** [comment]
這是去年十月的貼文
而我今年看AI展覽時 也有看到AI醫學檢測的攤位 稍微聊一下 
發覺他們已經開始跟各大醫院實施了 科技進展得真快

**Comment by BingRong1217** [comment]
AI結合醫療可以造福很多人，也讓更多人有活下去的機會

**Comment by limitx0** [comment]
AI在醫學影像判讀上應用好多
這樣未來醫生是不是就可以不用判讀影像了

**Comment by Aerofe** [comment]
如果這個技術能普及的話對醫療發展肯定幫助很大！

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by awkward-willy** [comment]
台灣身為醫療大國，在智慧醫療的部分一直不斷力求進步，透過機器學習來輔助X光判讀等功能是非常有用的，期望這些科技未來能延長人們的壽命、及早發現人們的健康問題，並減少醫護人員的負擔。

**Comment by larry40305** [comment]
AI能夠幫助我們更快速地做更多醫療辨識，這樣能夠避免醫療資源的不足。

**Comment by HsuChen88** [comment]
隨著這個軟體的出現，我的室友有救了，天天兩包菸，肺癌看不見。但若有這個輔助工具的話，相信很早就能發現肺癌並買好塔位。

**Comment by Bonus502530** [comment]
我想這項軟體的發明能夠帶給台灣人民蠻大的幫助，台灣的肺癌發生機率高居世界第15，身為南部的小孩上來北部讀書後很明顯的可以感受出南北部空氣品質的差異，我在南部時幾乎每天都擤鼻涕用掉的衛生非常可觀，但上來桃園讀書這樣的症狀減少非常多。不只是空氣品質台灣吸菸人口也蠻可觀，就算不抽菸身旁也有抽菸的人，因此我覺得這項技術對台灣的貢獻很大。

**Comment by ghost** [comment]
對於現在來說肺癌是一個很普遍的疾病，如果能透過AI及時抓出病灶並且分析，除了能減少醫師的工作量，更能使病人及早獲得治療
!
